package.json

```diff
@@ -55,6 +55,7 @@
     "json-stable-stringify": "^1.0.0",
     "levn": "^0.3.0",
     "lodash": "^4.0.0",
+    "minimatch": "^3.0.0",
```

Comment:
I realized that this is not going to work: `minimatch` actually doesn't behave the same way as `glob`, in that `glob` requires the _whole_ filename to match the pattern, whereas minimatch only checks that _any_ part of the filename matches. So an override rule that's supposed to match `**/test.js` is actually matching, e.g., `src/components/Component.test.jsx`. I'm wondering whether it would make sense to enumerate the list of files matching every glob pattern in `overrides` and then find the intersection between those files and the files matched from the CLI glob pattern. The downside is that there would definitely be a performance penalty. I'll benchmark and see how it compares.